platform/os_linux_test.go

```diff
@@ -0,0 +1,29 @@
+package platform
+
+import (
+	"testing"
+	"time"
+)
+
+// Default timeout in ExecuteCommand is 10 seconds so test should fail with 12 seconds execution time
+func TestExecuteCommandTimeout(t *testing.T) {
+	const timeout = 10 * time.Second
```

Comment:
can we have less timeout value so that tests won't hang for long?

Reply:
done